Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -929,6 +929,57 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Replicate Encrypted Properties wit
         }
     }
 }
+
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Pinned Certificate Failure - SGColl", "[.SyncServerCollection]") {
+    if(!Address::isSecure(_address)) {
+        _address = { kC4Replicator2TLSScheme,
+                    C4STR("localhost"),
+                    4984 };
+    }
+    REQUIRE(Address::isSecure(_address));
+
+    // Using an unmatched pinned cert:
+    pinnedCert =                                                               \
+        "-----BEGIN CERTIFICATE-----\r\n"                                      \
```

Comment:
I don't think you need back-slashes at end of each lines.

Reply:
It's just what I copied from the previous test. (Whereas the ones from the other test I copied from the cert file).